connect/api.go

```diff
@@ -57,3 +58,46 @@ func GetProduct(productQuery Product) (Product, error) {
 	}
 	return remoteProduct, nil
 }
+
+// updateSystemAPI resend the system's hardware info to SCC.
+// https://github.com/SUSE/connect/wiki/SCC-API-%28Implemented%29#update-system
+func updateSystemAPI(distroTarget, namespace string, instanceData []byte) error {
```

Comment:
Maybe putSystem() is better here.

Reply:
why not just `updateSystem()` ?

Reply:
btw, the link in the comment doesn't seem to be valid anymore

Reply:
> why not just `updateSystem()` ?

because I have already taken that name in client.go, so it needs to something different. I like putSystem()  here, and it follows a convention we already have with GetProduct() and GetActivations()

Reply:
> > why not just `updateSystem()` ?
> 
> because I have already taken that name in client.go, so it needs to something different. I like putSystem() here, and it follows a convention we already have with GetProduct() and GetActivations()

thanks - I should have check that

Reply:
I made some changes around here. Maybe better?

Reply:
looks good